### PR TITLE
FirmareControl/FrontPanel/SystemdConnector: moved TRACE_LX to TRACE()

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: SystemdConnector/SystemdConnector.cpp [hunk 1]
```diff
@@ -36,17 +36,17 @@ namespace Plugin {
...
+        TRACE(Trace::Information, (_T("Notify systemd that the Platform is up and running.\n")));
 
         int rc = sd_notifyf(0,
             "READY=1\n"
             "STATUS=Platform Server is Ready (from WPE Framework Compositor Plugin)\n"
             "MAINPID=%lu",
             ::getpid());
         if (rc) {
-            TRACE_L1("Notify Nexus Server Ready to systemd: FAILED (%d)\n", rc);
+            TRACE(Trace::Information, (_T("Notify Nexus Server Ready to systemd: FAILED (%d)\n"), rc));
```
Comment (pwielders) [inline SystemdConnector/SystemdConnector.cpp:47]: Maybe we should make this an Error ?

File: FileTransfer/FileTransfer.h [hunk 2]
```diff
@@ -161,7 +161,7 @@ namespace Core {
                     loop->second.Unregister(callback);
                     if (loop->second.HasCallbacks() == false) {
                         if (inotify_rm_watch(_notifyFd, index->second) < 0) {
-                            TRACE_L1(_T("Invoke of inotify_rm_watch failed"));
+                            TRACE(Trace::Information, (_T("Invoke of inotify_rm_watch failed")));
```
Comment (pwielders) [inline FileTransfer/FileTransfer.h:164]: Isn't this an Error ?

**Review by pwielders — CHANGES_REQUESTED** [review]

File: SystemdConnector/SystemdConnector.cpp (hunk 1 above)
Comment (HaseenaSainul) [reply SystemdConnector/SystemdConnector.cpp]: done

File: FileTransfer/FileTransfer.h (hunk 2 above)
Comment (HaseenaSainul) [reply FileTransfer/FileTransfer.h]: done

**Review by pwielders — APPROVED** [review]

**Pull request merged**